.github/workflows/release.yaml

```diff
@@ -42,5 +42,4 @@ jobs:
         git config --global tag.gpgsign true
         git config user.name "github-actions[bot]"
         git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
-        gh release create "$VERSION" --title "$VERSION" --notes "Release $VERSION" --draft --generate-notes
-        gitsign verify $(git rev-list --tags --max-count=1) --certificate-identity-regexp="https://github.com/${{ github.repository }}/*" --certificate-oidc-issuer="https://token.actions.githubusercontent.com"
+        gh release create "$VERSION" --title "$VERSION" --notes "Release $VERSION" --draft
```

Comment:
We aren't creating an annotated tag which is probably why the verification isn't working:
```
bash-5.2$ git tag -a --sign test_tag
Your browser will now be opened to:
...
tlog entry created with index: 105090273
bash-5.2$ git verify-tag test_tag
tlog index: 105090273
gitsign: Signature made using certificate ID 0xae04dcbbd9634103a143bf356b9d3025c99cba03 | CN=sigstore-intermediate,O=sigstore.dev
gitsign: Good signature from [...](https://github.com/login/oauth)
Validated Git signature: true
Validated Rekor entry: true
Validated Certificate claims: false
WARNING: git verify-commit does not verify cert claims. Prefer using `gitsign verify` instead.
```
versus:
```
bash-5.2$ git verify-tag v0.13.1
error: v0.13.1: cannot verify a non-tag object of type commit.
``